collect_app/src/main/java/org/odk/collect/android/utilities/MenuDelegate.java

```diff
@@ -8,7 +8,10 @@
  * Delegate responsible for the menu (options menu) of an {@link android.app.Activity}. Allows the
  * menu to be defined and tested separately from the Activity itself. Methods should be called
  * from their corresponding Activity lifecycle methods.
+ *
+ * @deprecated use {@link androidx.core.view.MenuProvider} instead
```

Comment:
`MenuProvider` basically gives us a more integrated and "official" version of what we were doing with `MenuDelegate`. Annoyingly, there's basically no docs for it yet other than the video it was introduced (as far as I can tell) in: https://www.youtube.com/watch?v=OE-tDh3d1F4.